tests/pytorch/test_recipe.py

```diff
@@ -23,6 +23,7 @@
 fp8_available, reason_for_no_fp8 = FP8GlobalStateManager.is_fp8_available()
 
 
+# FP8 per tensor delayed scaling
```

Comment:
Rather than comment maybe we could just change the name of the class to make that explicit?